templates/admin/repo/unadopted.tmpl

```diff
@@ -37,7 +37,7 @@
 											<input type="hidden" name="action" value="adopt">
 											<input type="hidden" name="q" value="{{$.Keyword}}">
 											<input type="hidden" name="page" value="{{$.CurrentPage}}">
-											{{template "base/modal_actions_confirm" (dict "locale" $.locale)}}
+											{{template "base/modal_actions_confirm" .}}
```

Comment:
No, don't do thing (passing the root data into a sub-template which doesn't need it)

Reply:
That's the way we do it in 46 other places...
but it should be possible to omit the `.` without consequences I think